From 888f52be7bf559fcb734615d95ba904ef175bd97 Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Wed, 23 Dec 2015 10:24:35 +0200
Subject: [PATCH 26/66] defconfig: add NAS related utils to armv8-le image

Enable -O2 and add some additional utilities needed for NAS

Change-Id: I162dfb59ebfe5af57b8f41c6bc064a667dfef40b
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/26258
Reviewed-by: Lior Amsalem <alior@marvell.com>
Reviewed-by: Shadi Ammouri <shadi@marvell.com>
Tested-by: Shadi Ammouri <shadi@marvell.com>
---
 configs/mvebu_armv8_le_defconfig | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/configs/mvebu_armv8_le_defconfig b/configs/mvebu_armv8_le_defconfig
index 0d9ec1c..f6f932e 100644
--- a/configs/mvebu_armv8_le_defconfig
+++ b/configs/mvebu_armv8_le_defconfig
@@ -1,4 +1,5 @@
 BR2_aarch64=y
+BR2_OPTIMIZE_2=y
 BR2_TOOLCHAIN_EXTERNAL=y
 BR2_MVEBU_RELEASE_NAME="2016_T1.0.eng_drop_v1"
 BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
@@ -73,11 +74,18 @@ BR2_PACKAGE_FILE=y
 BR2_PACKAGE_SCREEN=y
 BR2_PACKAGE_TIME=y
 BR2_PACKAGE_WHICH=y
+BR2_PACKAGE_ATTR=y
+BR2_PACKAGE_COREUTILS=y
 BR2_PACKAGE_FTOP=y
 BR2_PACKAGE_HTOP=y
 BR2_PACKAGE_IOTOP=y
 BR2_PACKAGE_KVMTOOL=y
+BR2_PACKAGE_PROCPS_NG=y
 BR2_PACKAGE_TAR=y
+BR2_PACKAGE_UTIL_LINUX_BINARIES=y
+BR2_PACKAGE_UTIL_LINUX_FSCK=y
+BR2_PACKAGE_UTIL_LINUX_MOUNT=y
+BR2_PACKAGE_UTIL_LINUX_RESET=y
 BR2_PACKAGE_LESS=y
 BR2_PACKAGE_VIM=y
 BR2_TARGET_ROOTFS_TAR_GZIP=y
-- 
1.9.1

